src/Events/Event.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Events;
+
+use Hyperdrive\Combat\Combat;
+use Hyperdrive\Combat\Enemies;
+use Hyperdrive\Geography\PlanetSurface;
+use Hyperdrive\HyperdriveNavigator;
+use Hyperdrive\Output\OutputContract;
+use Hyperdrive\Pilot\Pilot;
+use Hyperdrive\Quest\Quest;
+use Hyperdrive\Quest\QuestLog;
+use Hyperdrive\Ship\Ship;
+
+class Event
```

Comment:
Z tej klasy można by spokojnie zrobić kilka klas. Jedna klasa do zarządzania eventami ("EventManager", która by losowała, czy uruchamiała eventy), oraz n klas dziedziących po klasie bazowej Event (czy implementująca interfejs), które odpowiadały by za konkretny event i to, co się w nim dzieje.